packages/react/src/components/ImageCard/ImageHotspots.test.jsx

```diff
@@ -307,8 +307,9 @@ describe('ImageHotspots', () => {
           selectedHotspots={[]}
         />
       );
-
-      userEvent.click(screen.getByAltText(testImageText));
+      userEvent.click(screen.getByAltText(testImageText), {
+        ctrlKey: true,
+      });
       expect(onAddHotspotPosition).not.toHaveBeenCalled();
```

Comment:
This test looks like it is clicking with the ctrl button pressed but the assertion is that `onAddHotSpotPosition` has not been called. Shouldn't that be the opposite?

Reply:
in the first case the ImageHotspots isn't editable